core/templates/pages/topic-viewer-page/info-tab/topic-info-tab.directive.html

```diff
@@ -0,0 +1,140 @@
+<div class="oppia-topic-info-container">
+  <md-card class="oppia-page-card oppia-long-text">
+    <h2 class="tab-title"><[$ctrl.getTopicName()]></h2>
+    <div class="info-tab-content">
+      <div class="info-tab-left-container" ng-if="!$ctrl.isMobile">
+        <md-card class="info-tab-count">
+          <div style="padding-top: 12%;">
```

Comment:
Avoid inline styling here and elsewhere in this file.

Reply:
@aks681